fairlearn/reductions/_exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -22,7 +22,7 @@ def _mean_pred(X, hs, weights):
     return pred[weights.index].dot(weights)
 
 
-class ExponentiatedGradient(Reduction):
+class ExponentiatedGradient(BaseEstimator, ClassifierMixin):
     """An Estimator which implements the exponentiated gradient approach to reductions.
```

Comment:
This should be `BaseEstimator` and `MetaEstimatorMixin`. I don't mind having `ClassifierMixin` for now, but it will go away once we support `ExponentiatedGradient` with regression.

Reply:
I don't currently see `_estimator_type` and `score`, so I will leave them for another PR. Addressed your other comments, though!